wkcuber/image_readers.py

```diff
@@ -107,23 +117,54 @@ def read_channel_count(self, _file_name):
         return 1
 
 
-class ImageReader:
+class TiffImageReader(ImageReader):
+    def read_array(self, file_name, dtype, z_slice):
+        tif_file = TiffFile(file_name)
```

Comment:
In a recent script that I wrote, I had to transpose the image from `tifffile` because it uses inverted dimensions as compared to wkw. I can provide a test file.